### PR TITLE
Blog App: validations and model specs

### PR DESCRIPTION
**Pull request by kalbek**
## Changes made in this PR:
-Add the following validations:
 - User model:
   - Name must not be blank.
   - PostsCounter must be an integer greater than or equal to zero.
 - For the Post model:
   - Title must not be blank.
   - Title must not exceed 250 characters.
   - CommentsCounter must be an integer greater than or equal to zero.
   - LikesCounter must be an integer greater than or equal to zero.
- Added unit specs for all of your models' methods and validations.

### CODE REVIEW

File: test/models/post_test.rb
Comment (fobadara) [inline test/models/post_test.rb:1]: [OPTIONAL] Using a description block make it easier for other developers to understand your tests. I think you should do something like this:
```
describe 'validations' do
  it { should validate_presence_of(:name) }
end
```

File: test/models/user_test.rb
Comment (fobadara) [inline test/models/user_test.rb:1]: [OPTIONAL] Using a description block make it easier for other developers to understand your tests. I think you should do something like this:
```
describe 'validations' do
  it { should validate_presence_of(:name) }
end
```

File: README.md
Comment (fobadara) [inline README.md:1]: [REQUIRED] In your README file in the `key features` you have a long dash that is doing nothing. This makes your project look unprofessional. Kindly fix this:
image

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @kalbek, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Good first attempt
✅ Good file structure

## Required Changes ♻️
_Check the comments under the review._
I had to include some comments here because it could not be done inline
[REQUIRED]  I tried running your tests but I get errors when I use the commands you mentioned in your README.  Kindly fix this and tell us how to run your test properly in your README.
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢 🟢 🟢

Hi @kalbek, 

Good job working on the changes highlighted by the previous reviewer 👍🏽

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights:

✅ You used Gitflow correctly 👍🏽
✅ The Linter checks passed successfully 👍🏽
✅ Set validation and model specs 👍🏽
✅ All tests passed successfully 👍🏽

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**